pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1472,7 +1479,8 @@ func PublishEdgeviewToZedCloud(ctx *zedagentContext,
 	//We queue the message and then get the highest priority message to send.
 	//If there are no failures and defers we'll send this message,
 	//but if there is a queue we'll retry sending the highest priority message.
-	queueInfoToDest(ctx, dest, "global", buf, size, true, false, false,
+	// The first update is important, and that informs zedcloud the edgeivew is 'Active'
```

Comment:
This comment would make more sense on line 1454 together with an explanation that the counters are always zero when the first Evinfo is set. Alternatively make it explicitly track whether it is the first one it sends.

Reply:
makes sense. updated.